wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -257,7 +236,7 @@ void compile() throws IOException {
     }
 
     JavaGenerator javaGenerator = JavaGenerator.get(schema)
-        .withCustomProtoAdapter(protoTypeToJavaClassName, protoTypeToAdapterConstant)
+        .withProfile(profile)
         .withAndroid(emitAndroid)
```

Comment:
sadly no . . . this change hasn’t yet promoted the `android` flag into the Profile. That’ll come soon.